herbert_core/utilities/classes/@unique_references_container/unique_references_container.m

```diff
@@ -342,6 +342,14 @@
     methods % overloaded indexers, subsets, find functions
         
         function field_vals = get_unique_field(self, field)
+        %GET_UNIQUE_FIELD each of the unique objects referred to by self
+        % should have a property named 'field'. The code below takes each of the
+        % referred objects in turn, extracts the object referred to by
+        % 'field' and stores it in the unique_OBJECTS_container field_vals
+        % created here. field_vals will then contain unique copies of all
+        % the values of 'field' within the objects referred to in self, indexed
```

Comment:
What happens if any of the `field` values are the same? Are those eliminated and the remainder collapsed down?

Reply:
@oerc0122 I am unclear what scenario you have in mind. The fields will be actual properties of the classes in the container and will be unique for each stored class object. Thus a fermi chopper instrument will have a field 'moderator' and this method will coy all the moderators and store them in a unique_objects_container indexed by the original instrument indices. So I do not know what 'same' means here